examples/cm6-graphql-parcel/src/index.ts

```diff
@@ -8,6 +8,7 @@ const state = EditorState.create({
   extensions: [basicSetup, StreamLanguage.define(graphql)],
 });
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const view = new EditorView({
```

Comment:
I don't know the purpose for assigning the instance to a variable, maybe the following is better
```suggestion
new EditorView({
```